src/components/checkbox/checkbox.test.js

```diff
@@ -238,27 +238,21 @@ context("Testing Checkbox component", () => {
   it("should render Checkbox component with error message", () => {
     CypressMountWithProviders(<CheckboxComponent error="Error has occurred" />);
 
-    checkboxIcon()
-      .should("have.attr", "aria-label", "Error has occurred")
-      .should("have.attr", "data-element", "error");
+    checkboxIcon().should("have.attr", "data-element", "error");
```

Comment:
Personally I think we still need to verify if the tooltip message has been rendered alongside checking the icon. What do you think about adding the following assertion so we are testing if the icon and tooltip are both rendered? This would also apply to the two succeeding tests as well :) 

```suggestion
    checkboxIcon().should("have.attr", "data-element", "error");
    tooltipPreview().should("have.text", "Error has occurred");
```

Reply:
this was not testing that the tooltip was rendered, this was just checking if the aria-label attribute is on the icon, which should not be added here, so I decided to remove it completely. As for the tooltip message, I'll leave it to QA, as this should be added later by them if necessary:
![image](https://user-images.githubusercontent.com/22885392/223105180-05064317-b665-4972-8fdf-8eba7f11b37b.png)

Reply:
We'll need to make sure that the person QA'ing this definitely does add the needed Cypress tests.

Reply:
I don't agree with adding additional tests because we:
1. already have tests for tooltip for each component 
2. need to unskip all tests with `validation`

Reply:
Good point @DlgSHi 👍🏼

In that case what do you think @grabkowski of amending the test descriptions to something like "should render Checkbox component with error icon" - since we aren't asserting if the message has been rendered but the icon. Happy to accept if otherwise as this is minor

Reply:
changed